UmatiServerLib/Instantiation.cpp

```diff
@@ -42,12 +36,10 @@ open62541Cpp::UA_NodeId readDataType(UA_Server *pServer, open62541Cpp::UA_NodeId
   return ret;
 }
 
-UA_Int32 readValueRank(UA_Server *pServer, open62541Cpp::UA_NodeId nodeId)
-{
+UA_Int32 readValueRank(UA_Server *pServer, open62541Cpp::UA_NodeId nodeId) {
   UA_Int32 ret;
   UA_StatusCode status = UA_Server_readValueRank(pServer, *nodeId.NodeId, &ret);
-  if (status != UA_STATUSCODE_GOOD)
-  {
+  if (status != UA_STATUSCODE_GOOD) {
     std::stringstream ss;
     ss << "Could not read DataType from NodeId (" << static_cast<std::string>(nodeId) << "), Error: " << UA_StatusCode_name(status);
```

Comment:
Should be _ValueRank_ instead of _Datatype_?

```suggestion
    ss << "Could not read ValueRank from NodeId (" << static_cast<std::string>(nodeId) << "), Error: " << UA_StatusCode_name(status);
```